sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/db/SponsorGroupDbItem.kt

```diff
@@ -0,0 +1,5 @@
+package co.touchlab.sessionize.db
+
+import co.touchlab.droidcon.db.Sponsor
+
+data class SponsorGroupDbItem(val groupName: String, val sponsors: List<Sponsor>)
```

Comment:
I think these data objects need a rename, but we should do that post merge (and "to what" I'm not sure, but seems a little verbose)